docs/api/ref/Icon.md

```diff
@@ -0,0 +1,55 @@
+---
+id: Icon
+sidebar_label: Icon
+title: Icon
+hide_title: true
+---
+# `Icon`
+
+```typescript
+interface Icon {
+  src: string;
+  size: string;
+  type: string;
+}
+```
+
+App Metadata icon's description. 
+
+Various properties may be used by the Desktop Agent to decide which icon is the most suitable to be used considering the application chooser UI, device DPI and formats supported by the system.
+
+#### Example
+
+```js
+"icons": [
+  {
+    "src": "https://app.foo.icon/app_icons/lowres.webp",
+    "size": "48x48",
+    "type": "image/webp"
+  },
+  {
+    "src": "https://app.foo.icon/app_icons/hd_hi.svg",
+    "size": "72x72"
+  }
+]
+```
+
+## Properties
+
+### `src`
+
+The fully qualified url to the icon.
```

Comment:
Please note that in the PWA manifest spec the url may be relative. However in the context of FDC3 I couldn't find where a Desktop agent may retrieve the base url of an application so that when given a relative url it could construct the fully qualified url.
That is the reason why only fully qualified url are allowed here.